src/tests/catwalk_tests/test_baselines.py

```diff
@@ -80,6 +82,14 @@ def test_predict_proba(self):
             np.testing.assert_array_equal(results, expected_results)
 
 
+@pytest.mark.parametrize('operator', [">", ">=", "<", "<=", "=="])
```

Comment:
This requires the whitelist of operators to be defined in two places, it looks like. I would prefer the operator_lookup in the tested function to be just a module-level constant, and you use the dictinary's `.keys()` to get the list of things you need to test, which is now much harder to make obsolete.

Reply:
Yeah, I didn't want to go too far commenting on the workings of that method, but I would agree, and say...

It would be nicer for the `operator_lookup` dictionary to *not* be created on every invocation of `get_operator_method`, and instead that it be static, at the module level, and therefore accessible to this test 😸 

So for example in `thresholders.py`:

```
OPERATOR_METHODS = {...}

def get_operator_method(...):
    try:
        operator_method = OPERATOR_METHODS[operator_string]
    ...
```

...And then, indeed, the test can just be like:

```
@pytest.mark.parametrize('operator', tuple(OPERATOR_METHODS))
def ...
```

Reply:
It's always worth going too far in the name of improvement :)